api-service/controller/oracle_database_agreements.go

```diff
@@ -90,6 +90,11 @@ func (ctrl *APIController) UpdateOracleDatabaseAgreement(w http.ResponseWriter,
 		return
 	}
 
+	if req.Unlimited && !req.CatchAll{
+		utils.WriteAndLogError(ctrl.Log, w, http.StatusBadRequest, utils.NewError(errors.New("Error")))
```

Comment:
Instead of 
```
utils.NewError(errors.New("Error")))
```
we should put something meaningful!

1. Try utils.NewErrorf to create an error from a string directly when you don't have an error
2. Insert a meaningful message: "Agreement is unlimited so it must be even CatchAll"...

Reply:
> Instead of
> 
> ```
> utils.NewError(errors.New("Error")))
> ```
> 
> we should put something meaningful!
> 
>     * [x]  Try utils.NewErrorf to create an error from a string directly when you don't have an error
> 
>     * [x]  Insert a meaningful message: "Agreement is unlimited so it must be even CatchAll"...